### ISSUE TITLE
"dnx ef database update" step fails with "Unable to find an entry point named 'sqlite3_close_v2' in DLL 'sqlite3'."

### ISSUE THREAD

**Issue by klothe** [aspnet/EntityFramework.Docs#110]
I encountered this error by following the instructions under ["Getting Started on OSX"](https://ef.readthedocs.org/en/latest/getting-started/osx.html) using Mac OS 10.9.5 and dnx/dnu 1.0.0-rc1-16231. All went well until trying to apply database migrations:
https://ef.readthedocs.org/en/latest/getting-started/osx.html#create-your-database
```
$ dnx ef database update
System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_close_v2' in DLL 'sqlite3'.
   at Microsoft.Data.Sqlite.Interop.NativeMethods.sqlite3_close_v2(IntPtr db)
   at Microsoft.Data.Sqlite.Interop.Sqlite3Handle.ReleaseHandle()
   at System.Runtime.InteropServices.SafeHandle.InternalDispose()
   at System.Runtime.InteropServices.SafeHandle.Dispose(Boolean disposing)
   at System.Runtime.InteropServices.SafeHandle.Dispose()
   at Microsoft.Data.Sqlite.SqliteConnection.Close()
   at Microsoft.Data.Entity.Storage.RelationalConnection.Close()
   at Microsoft.Data.Entity.Storage.Internal.SqliteRelationalConnection.Close()
   at Microsoft.Data.Entity.Storage.Internal.RelationalCommand.Execute[T](IRelationalConnection connection, Func`3 action, String executeMethod, Boolean openConnection, Boolean closeConnection)
   at Microsoft.Data.Entity.Migrations.HistoryRepository.Exists()
   at Microsoft.Data.Entity.Migrations.Internal.Migrator.Migrate(String targetMigration)
   at Microsoft.Data.Entity.Design.MigrationsOperations.UpdateDatabase(String targetMigration, String contextType)
   at Microsoft.Data.Entity.Commands.Program.Executor.Execute(Action action)
Unable to find an entry point named 'sqlite3_close_v2' in DLL 'sqlite3'.
```
This seems like a version mismatch, but I don't know where. [SQLite itself added "sqlite3_close_v2" in 3.7.14 in 2012](https://www.sqlite.org/releaselog/3_7_14.html). Mac OS X 10.9.5 comes with SQLite 3.7.13, but I'm not sure if the version of SQLite included with the OS is even related.

Here the versions of packages that I have (other than SQLite and ones specified by the project.json):
```
$ dnx --version
Microsoft .NET Execution environment
 Version:      1.0.0-rc1-16231
 Type:         CoreClr
 Architecture: x64
 OS Name:      Darwin
 OS Version:   10.9
 Runtime Id:   osx.10.9-x64
$ dnu --version
Microsoft .NET Development Utility
 Version:      1.0.0-rc1-16231
 Type:         CoreClr
 Architecture: x64
 OS Name:      Darwin
 OS Version:   10.9
 Runtime Id:   osx.10.9-x64
$ dnvm --version
1.0.0-rc2-15546
```
Suggestions would be appreciated!

**Comment by natemcmaster** [comment]
You need to run on OS X Yosemite or newer. (It's listed as a prerequisite in https://ef.readthedocs.org/en/latest/getting-started/osx.html#prerequisites.) OSX Mavericks is not supported by CoreCLR.

**Comment by klothe** [comment]
Thanks!

**Comment by raholland79** [comment]
I'm on Yosemite and get this.

Same version output as the OP - except the OS is 10.10.5

**Comment by klothe** [comment]
By the way, this was fixed by upgrading to Mac OS 10.11 with SQLite 3.10.1.
